disconnected/about.adoc

```diff
@@ -0,0 +1,61 @@
+Without a direct internet connection, you must perform additional setup and configuration for your cluster to maintain full functionality in the disconnected environment.
+
+[id="glossary-disconnected_{context}"]
+== Glossary of disconnected environment terms
+
+Although it is used throughout the {product-title} documentation, _disconnected environment_ is a broad term that can refer to environments with various levels of internet connectivity.
+Other terms are sometimes used to refer to a specific level of internet connectivity, and these environments might require additional unique configurations.
+
+The following table describes the different terms used to refer to environments without a full internet connection:
+
+.Disconnected environment terms
+[cols=".^2,.^4",options="header"]
+|====
+|Term |Description
+
+|Air-gapped network
+|An environment or network that is completely isolated from an external network.
+
+This isolation depends on a physical separation, or an "air gap", between machines on the internal network and any other part of an external network.
+Air-gapped environments are often used in industries with strict security or regulatory requirements.
+
+|Disconnected environment
+|An environment or network that has some level of isolation from an external network.
+
+This isolation could be enabled by physical or logical separation between machines on the internal network and an external network.
+Regardless of the level of isolation from the external network, a cluster in a disconnected environment does not have access to public services hosted by Red{nbsp}Hat and requires additional setup to maintain full cluster functionality.
+
+|Restricted Network
+|An environment or network with limited connection to an external network.
+
+A physical connection may exist between machines on the internal network and an external network, but network traffic is limited by additional configurations, such as with firewalls and proxies.
+|====
+
+[id="preferred-methods_{context}"]
+== Preferred methods for working with disconnected environments
+
+// Not sure about the heading above, I'm trying to find some phrasing that basically says "You should probably use these things if you don't have a strong reason to go with other options".
+
+You can choose between multiple options for most aspects of managing a cluster in a disconnected environment.
+For example, when mirroring images you can choose between using the oc-mirror {oc-first} plugin or using the `oc adm` command.
+
+However, some options provide a simpler and more convenient user experience for disconnected environments, and are the preferred method over their alternatives.
+
+Unless your organizational needs require you to choose another option, use the following methods for mirroring images, installing your cluster, and updating your cluster:
```

Comment:
Can I think of `Restricted Network` is a type of  `Disconnected environment`? If yes, maybe firewalls and proxies is some alternative method to mirror images. But from the document here, sounds like whatever "Air-gapped network" or "Restricted Network" we always recommend the mirror-images methods.

Reply:
And another issue found here is, actually in our current doc, such as for https://docs.openshift.com/container-platform/4.17/disconnected/installing.html , in the title we name the installation method that requires mirror image as `Installing a cluster in a disconnected environment `, but in the following chapters, they're all saying restricted network, like:
```
[Installing a cluster on Azure in a restricted network](https://docs.openshift.com/container-platform/4.17/installing/installing_azure/ipi/installing-restricted-networks-azure-installer-provisioned.html#installing-restricted-networks-azure-installer-provisioned)
```

Reply:
@gpei thanks for calling this out, I am standardizing all the names to "disconnected environment" in https://issues.redhat.com/browse/OSDOCS-11846

Reply:
"Restricted network" isn't a term that users use but that we (RH) added to be precise with the language, but for clarity we want to use "disconnected networks". In this last link from Seb we can see the term "partially disconnected network". We could optionally use it to refer to restricted networks too. Wrt giving examples IIRC you already wrote an intro mentioning that restricted networks make use of proxies or firewalls so I don't think we have to do again here.